fence-agents.spec.in

```diff
@@ -65,6 +65,7 @@ fence-agents-intelmodular \\
 fence-agents-ipdu \\
 fence-agents-ipmilan \\
 fence-agents-kdump \\
+fence-agents-kubevirt \\
```

Comment:
Let's remove it from here (as a dependency of the -all subpackage), to avoid pulling in the additional libraries for users who install fence-agents-all.

Reply:
Removed